app/test_split_integer.py

```diff
@@ -2,20 +2,35 @@
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
-    pass
+    value = 32
```

Comment:
These variables are extra
```
assert sum(split_integer(32, 6)) == 32
```
You can use just one expression which is easier to read

Reply:
Ok. I was just trying to write it in detail, as the tutorial video said.